src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -509,7 +508,7 @@ namespace BaseboardElectric {
             CompName = baseboard->Baseboard(BaseboardNum).EquipName;
             state.dataSize->DataFracOfAutosizedHeatingCapacity = 1.0;
             state.dataSize->DataZoneNumber = baseboard->Baseboard(BaseboardNum).ZonePtr;
-            SizingMethod = HeatingCapacitySizing;
+            SizingMethod = static_cast<int>(AutoSizingType::HeatingCapacitySizing);
```

Comment:
Again, most of what you will see in this particular commit is moving away from the duplicate ints out in DataHVACGlobals into the official enumeration in Autosizing/Base.hh.